feature-pack/src/main/resources/modules/system/layers/base/org/jboss/weld/core/main/module.xml

```diff
@@ -52,6 +51,6 @@
         <module name="org.jboss.weld.api" export="true"/>
         <module name="org.jboss.weld.spi" export="true"/>
         <module name="org.jboss.logging" />
-        <module name="sun.jdk" optional="true" />
+        <module name="jdk.unsupported"/>
```

Comment:
JDeps reports you need the following additional modules:

* ``java.desktop``
* ``java.logging``
* ``java.naming``
* ``java.xml``